amsterdam/Issues/taxiZoneExemption/description.xml

```diff
@@ -0,0 +1,60 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo Taxi zone exemption Amsterdam</en>
+        <nl>Demo Taxizone-ontheffing Amsterdam</nl>
+    </Name>
+    <ShortName>
+        <en>Demo Taxi zone exemption Amsterdam</en>
+        <nl>Demo Taxizone-ontheffing Amsterdam</nl>
+    </ShortName>
+    <SchemeManager>irma-demo</SchemeManager>
+    <IssuerID>amsterdam</IssuerID>
+    <CredentialID>taxiZoneExemption</CredentialID>
+    <Description>
+        <en>Taxi zone exemption Amsterdam</en>
+        <nl>Taxizone-ontheffing Amsterdam</nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
```

Comment:
Are you sure you want to have this if you also have a `validFrom` and a `validUntil`? This flag enforces that you cannot have two credentials of this instance at the same time in your IRMA app.

Reply:
For the time being we want this.
The `validFrom` and `validUntil` are for now just for display / informational purposes.
For the future we do expect to drop the singleton requirement, but that might not happen for another year.
For your information, this exemption will typically be valid for 2 years.

Or would it cause issues if we change the credentialType to not be a singleton in the future?

Reply:
It should be possible to change this later if I'm right. I thought, if you know it for sure now that it's needed, then you can also set it correctly now. But if you want to think this a bit more through, then that's also fine and then we keep it `true` for now.